<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>888</width>
        <height>131</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>72</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>LOGIN    </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_username">
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>USERNAME:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_password">
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PASSWORD:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_view">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>View.png</normaloff>View.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_back">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QPushButton" name="pushButton_login">
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
